app/dashboards/workshop_dashboard.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'administrate/base_dashboard'
+
+class WorkshopDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::Number,
+    kind: Field::SelectBasic.with_options(choices:
+      %w[
```

Comment:
Please fetch data using method 'kinds' of model Workshop (Workshop.kinds)

Reply:
Okay Fabian, but why kind should be plural? Names of table fields are singular.

Reply:
If you want fetch a list of defined enum you use plural name of field (kinds):

```
2.4.2 :002 > ap Workshop.kinds
{
                    "audi" => 0,
                 "citroen" => 1,
                    "fiat" => 2,
                   "honda" => 3,
                 "hyundai" => 4,
                   "mazda" => 5,
                    "opel" => 6,
                 "peugeot" => 7,
                 "renault" => 8,
                  "toyota" => 9,
              "volkswagen" => 10,
    "warsztat_lakierniczy" => 11,
    "warsztat_samochodowy" => 12
}
 => nil 
2.4.2 :003 >
```
Or in your case you can probably use: 
```
2.4.2 :004 > Workshop.kinds.keys
 => ["audi", "citroen", "fiat", "honda", "hyundai", "mazda", "opel", "peugeot", "renault", "toyota", "volkswagen", "warsztat_lakierniczy", "warsztat_samochodowy"] 
2.4.2 :005 >
```

Reply:
Something like this...
``` kind: Field::SelectBasic.with_options(choices: Workshop.kinds.keys) ```